libopenage/gui/guisys/private/platforms/context_extraction_cocoa.mm

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2016 the openage authors. See copying.md for legal info.
+// Copyright 2015-2017 the openage authors. See copying.md for legal info.
```

Comment:
That's a valid question. With current approach there is probably no need to do ObjC there.